pkg/config/components/honeycombExporter.yaml

```diff
@@ -0,0 +1,35 @@
+name: ToHoneycomb
+kind: HoneycombExporter
+summary: Export traces to Honeycomb in Honeycomb's event format
+description: |
+  This component exports traces to Honeycomb in Honeycomb's event format.
+ports:
+  - name: Traces
+    direction: input
+    type: HoneycombTraces
+properties:
+  - name: API_Key
+    summary: The API key to use to authenticate with Honeycomb
+    description: |
+      The API key to use to authenticate with Honeycomb.
+    type: string
+    validations: [nonblank]
+    default: ${HONEYCOMB_TRACES_APIKEY}
```

Comment:
This was deliberate to avoid confusion with API keys you may be using for other purposes; it's not a world where one honeycomb key makes sense anymore. I think it's likely that you'll have multiple Honeycomb keys, so I've changed the name to make it even clearer.